one_collect/src/helpers/exporting/scripting.rs

```diff
@@ -1000,9 +1003,20 @@ impl ScriptedUniversalExporter {
     pub fn from_script(
         self,
         script: &str) -> anyhow::Result<UniversalExporter> {
+        info!("Starting script execution: length={}", script.len());
+        debug!("Script content preview: {}", 
```

Comment:
Please log the entire script at INFO level instead of what's currently happening here.